app/onboarding-2.tsx

```diff
@@ -0,0 +1,231 @@
+import { Colors } from '@/constants/colors'
+import { useOnboardingStore } from '@/store/onboardingStore'
+import { router } from 'expo-router'
+import React from 'react'
+import {
+  Dimensions,
+  Pressable,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+
+const { width } = Dimensions.get('window')
+
+// Datos de los géneros
+const genres = [
+  { id: 1, name: 'Action' },
+  { id: 2, name: 'Adventure' },
+  { id: 3, name: 'Comedy' },
+  { id: 4, name: 'Drama' },
+  { id: 5, name: 'Horror' },
+  { id: 6, name: 'Sci-Fi' },
+  { id: 7, name: 'Thriller' },
+  { id: 8, name: 'Romance' },
+  { id: 9, name: 'Animation' },
+  { id: 10, name: 'Documentary' },
+  { id: 11, name: 'Fantasy' },
+  { id: 12, name: 'Mystery' },
+]
+
+export default function Onboarding2Screen() {
+  const { selectedGenres, setSelectedGenres, setCompleted } =
+    useOnboardingStore()
+
+  // Layout flexible para géneros
+  const containerPadding = 40 // 20px * 2 (izquierda y derecha)
+  const availableWidth = width - containerPadding
+
+  const toggleGenre = (genreId: number) => {
+    const newSelection = selectedGenres.includes(genreId)
+      ? selectedGenres.filter(id => id !== genreId)
+      : [...selectedGenres, genreId]
+    setSelectedGenres(newSelection)
+  }
+
+  const handleNext = () => {
+    if (selectedGenres.length >= 3) {
```

Comment:
There is no restriction in the design that indicate at least 3 genres needs to be selected, only one is enough